hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/AsyncLoggerSet.java

```diff
@@ -206,197 +194,114 @@ List<AsyncLogger> getLoggersForTests() {
       long segmentTxId, long firstTxnId, int numTxns, byte[] data) {
     Map<AsyncLogger, ListenableFuture<Void>> calls = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      ListenableFuture<Void> future = 
-        logger.sendEdits(segmentTxId, firstTxnId, numTxns, data);
-      calls.put(logger, future);
-    }
+    loggers.forEach(l -> calls.put(l, l.sendEdits(segmentTxId, firstTxnId, numTxns, data)));
     return QuorumCall.create(calls);
   }
 
   public QuorumCall<AsyncLogger, GetJournaledEditsResponseProto>
   getJournaledEdits(long fromTxnId, int maxTransactions) {
-    Map<AsyncLogger,
-        ListenableFuture<GetJournaledEditsResponseProto>> calls
-        = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      ListenableFuture<GetJournaledEditsResponseProto> future =
-          logger.getJournaledEdits(fromTxnId, maxTransactions);
-      calls.put(logger, future);
-    }
+    Map<AsyncLogger, ListenableFuture<GetJournaledEditsResponseProto>> calls = Maps.newHashMap();
+    loggers.forEach(l -> calls.put(l, l.getJournaledEdits(fromTxnId, maxTransactions)));
     return QuorumCall.create(calls);
   }
 
   public QuorumCall<AsyncLogger, RemoteEditLogManifest> getEditLogManifest(
       long fromTxnId, boolean inProgressOk) {
-    Map<AsyncLogger,
-        ListenableFuture<RemoteEditLogManifest>> calls
-        = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      ListenableFuture<RemoteEditLogManifest> future =
-          logger.getEditLogManifest(fromTxnId, inProgressOk);
-      calls.put(logger, future);
-    }
+    Map<AsyncLogger, ListenableFuture<RemoteEditLogManifest>> calls = Maps.newHashMap();
+    loggers.forEach(l -> calls.put(l, l.getEditLogManifest(fromTxnId, inProgressOk)));
     return QuorumCall.create(calls);
   }
 
-  QuorumCall<AsyncLogger, PrepareRecoveryResponseProto>
-      prepareRecovery(long segmentTxId) {
-    Map<AsyncLogger,
-      ListenableFuture<PrepareRecoveryResponseProto>> calls
-      = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      ListenableFuture<PrepareRecoveryResponseProto> future =
-          logger.prepareRecovery(segmentTxId);
-      calls.put(logger, future);
-    }
+  QuorumCall<AsyncLogger, PrepareRecoveryResponseProto> prepareRecovery(long segmentTxId) {
+    Map<AsyncLogger, ListenableFuture<PrepareRecoveryResponseProto>> calls = Maps.newHashMap();
+    loggers.forEach(l -> calls.put(l, l.prepareRecovery(segmentTxId)));
     return QuorumCall.create(calls);
   }
 
-  QuorumCall<AsyncLogger,Void>
-      acceptRecovery(SegmentStateProto log, URL fromURL) {
-    Map<AsyncLogger, ListenableFuture<Void>> calls
-      = Maps.newHashMap();
-    for (AsyncLogger logger : loggers) {
-      ListenableFuture<Void> future =
-          logger.acceptRecovery(log, fromURL);
-      calls.put(logger, future);
-    }
+  QuorumCall<AsyncLogger,Void> acceptRecovery(SegmentStateProto log, URL fromURL) {
```

Comment:
QuorumCall<AsyncLogger,Void> acceptRecovery(SegmentStateProto log, URL fromURL) {:25: ',' is not followed by whitespace. [WhitespaceAfter]